File: Logic Building/2_LOOPING & PATTERNS/4_Pattern Printing/16/16.go
// Q16: Print alphabet sequence (A, AB, ABC, ABCD).
// Input: An integer n
// Output: Alphabet sequence pattern

package main

import "fmt"

func main() {
    var n int
    fmt.Scan(&n)
    for i := 0; i < n; i++ {
        row := ""
        for j := 0; j <= i; j++ {
            row += string(rune('A' + j))
        }
        fmt.Println(row)
    }
}

File: Logic Building/2_LOOPING & PATTERNS/4_Pattern Printing/16/16.py
# Q16: Print alphabet sequence (A, AB, ABC, ABCD).
# Input: An integer n
# Output: Alphabet sequence pattern

n = int(input())
for i in range(n):
    for j in range(i + 1):
        print(chr(65 + j), end="")
    print()
